memphis/memphis.py

```diff
@@ -257,8 +261,12 @@ async def producer(self, station_name, producer_name, generate_random_suffix=Fal
             station_name_internal = get_internal_name(station_name)
             await self.start_listen_for_schema_updates(station_name_internal, create_res['schema_update'])
 
-            if self.schema_updates_data[station_name_internal] != {} and self.schema_updates_data[station_name_internal]['type'] == "protobuf":
-                self.parse_descriptor(station_name_internal)
+            if self.schema_updates_data[station_name_internal] != {}:
+                if self.schema_updates_data[station_name_internal]['type'] == "protobuf":
+                    self.parse_descriptor(station_name_internal)
+                elif self.schema_updates_data[station_name_internal]['type'] == "graphql":
+                    self.schema_graphql[station_name_internal] = build_schema(
+                        self.schema_updates_data[station_name_internal]['active_version']['schema_content'], assume_valid=True)
```

Comment:
what does it means - assume_valid?

Reply:
I deleted this option assume_valid - it is not relevant here